### ISSUE TITLE
Is it mac supported

### ISSUE THREAD

**Issue by RedRavenYouTube** [HJfod/GDShare#13]
does it work on Mac?

**Comment by ghost** [comment]
Yes, you can install it by adding [Geode](https://geode-sdk.org/) to your Geometry Dash, then you can add it using the Download tab in the Geode menu.

After installing Geode, click the Geode button on the main menu.
![image](https://user-images.githubusercontent.com/93995622/232158078-bcdb74f6-a4f7-486c-a235-334d6b1a59aa.png)
Now, select the "Download" tab.
![image](https://user-images.githubusercontent.com/93995622/232158136-25dbfe9c-b26e-4b5d-94bd-af2d30c3d0aa.png)
Look for "GDShare", then select "View"
![image](https://user-images.githubusercontent.com/93995622/232158229-6c7194ec-5074-45ad-80f6-5905bba40f46.png)
Click "Install"
![image](https://user-images.githubusercontent.com/93995622/232158258-39809dd6-512e-492a-8575-024ce89aa203.png)

Now, it should be installed. If the button doesn't appear on an online level screen, then restart your game.

**Comment by ghost** [comment]
Hope it helps

**Comment by RedRavenYouTube** [comment]
No. You can't use ModLoader3 with Geode for some reason. And I can't figure out how to use it.

**Comment by RedRavenYouTube** [comment]
ModLoader 3 is required for MegaHack

**Comment by ghost** [comment]
I don't use Mac, so I can't really provide much information. I'm pretty sure there can be conflict with other mod loaders, due to hooks and Geode's modifications.

**Comment by altalk23** [comment]
Yes it is supported

**Comment by FireDasher** [comment]
It has Mac support but the devs are too lazy to build it for Mac
